run-test.py

```diff
@@ -881,6 +881,36 @@ def gen_certs():
         scp_to(pod, "/tmp/insolar-jepsen-configs/reusekeys/discovery/*",
                INSPATH+"/scripts/insolard/reusekeys/discovery/")
 
+from queue import Queue
```

Comment:
sudden queuing bothers me, seems a bit risky, race-y. Also, this is not related to the original task, as far as i know.

Reply:
ok, let's simplify your life